src/test/java/catan/controllers/FunctionalTestUtil.java

```diff
@@ -1,6 +1,9 @@
+// TODO: remove checking of statusCode in each method. (Needs to be approved)
+
 package catan.controllers;
```

Comment:
I think we can remove it if we have assertion of object returned. In case of 'void' method, when there is no body, and nothing to assert, we should assert statusCode to make sure that is no errors and server replied with OK status